src/assets/scss/_variables.scss

```diff
@@ -138,3 +138,11 @@ $box-shadow-popover: 0 5px 10px rgba(0, 0, 0, 0.2);
 
 // Figma Variables
 $m3-elevation-light-3: 0px 4px 8px 3px rgba(0, 0, 0, 0.15), 0px 1px 3px 0px rgba(0, 0, 0, 0.30);
+
+// GeojsonMap based on Webview Variables
+$geojson-map-spacing: 10px;
+$geojson-map-background-color: #fff;
+$geojson-map-radius: 6px;
+$geojson-map-border-color: #cbd5e1;
+$geojson-map-muted-background-color: #F1F5F9;
+$geojson-map-z-index: 1;
```

Comment:
Since these variables are only used in `GeojsonMap`, they can just be defined locally there. They don't even need the `$geojson-map` prefix: if they're defined `GeojsonMap`, they'd be locally scoped there. Most of the variables are here because they're referenced in multiple components and can't be defined locally.